ui/src/views/Clients/ViewClients/index.jsx

```diff
@@ -146,7 +134,6 @@ export default class ViewClients extends PureComponent {
             disabled={loading}
             onSubmit={this.handleClientSearchSubmit}
             onChange={this.handleClientSearchChange}
-            defaultValue={searchDefaultValue}
             value={value}
```

Comment:
Removed defaultValue along with componentDidMount(). These were initially set to display a filled in searchbar if user reloads web page (e.g. http://localhost:5080/auth/clients?search=email%2Fbastien will have email/bastien in the searchbar).

However, this will be misleading since handleClientSearchSubmit does not take into account defaultValue and will assume empty string.